src/connectors/configure/connectConfigure.ts

```diff
@@ -129,6 +132,16 @@ const connectConfigure: ConfigureConnector = function connectConfigure(
         return getInitialSearchParameters(state, widgetParams);
       },
 
+      getWidgetRenderState(renderState) {
+        return {
+          ...renderState,
+          configure: {
```

Comment:
you can have multiple configure widgets in the same tree right? How does this behave? Not sure how to get a key for the configure though. The refine will be the same for them, but widgetParams will be different.

Reply:
Good call—this is why accumulating is more powerful than naively merging render states at the index level.

I'd suggest taking care of shared render state in future PRs, wdyt?

Reply:
fine for me if you write it down somewhere so we won't need to go through all PRs to find the unresolved questions afterwards :)